test/unit/core/simd/simd_test.cpp

```diff
@@ -9,44 +9,44 @@
 
 #include <seqan3/core/simd/all.hpp>
 
-using namespace seqan3;
-
-using int16x_t = seqan3::simd::simd_type<int16_t>::type;
-
 TEST(simd, auto_length)
 {
-    if constexpr(simd_traits<int16x_t>::max_length == 64)
+    using int16x_t = seqan3::simd::simd_type<int16_t>::type;
```

Comment:
can you not use using seqan3::simd within the test body? It spoils the entire test. Or is this a general problem?

Reply:
you mean `using namespace seqan3::simd`? sure I can xD

Reply:
We fully specify everything in tests now :)

Reply:
😭